tutorial/versioning/revisions.rst

```diff
@@ -0,0 +1,253 @@
+Revisions
+=========
+
+This sections introduces how doing modifications to a given recipe or source code without explicitly
+creating new versions, will still internally track those changes with a mechanism called revisions.
+
+
+Creating different revisions
+----------------------------
```

Comment:
Maybe we should specify that all these revisions are recipe revisions through all the tutorial until we get to the package revisions part?
```suggestion
Creating different recipe revisions
----------------------------------
```